src/systems/ec/implementations/vanilla/packages/ec-component-form-select/ec-component-form-select.js

```diff
@@ -0,0 +1,289 @@
+    checkbox.classList.add('ecl-checkbox');
+    checkbox.setAttribute('data-select-multiple-value', text);
+    const input = document.createElement('input');
+    input.classList.add('ecl-checkbox__input');
+    input.setAttribute('type', 'checkbox');
+    input.setAttribute('id', `${scope}-${id}`);
+    checkbox.appendChild(input);
+    const label = document.createElement('label');
+    label.classList.add('ecl-checkbox__label');
+    label.setAttribute('for', `${scope}-${id}`);
+    const box = document.createElement('span');
+    box.classList.add('ecl-checkbox__box');
+    box.appendChild(this.createCheckboxIcon());
+    label.appendChild(box);
+    label.appendChild(document.createTextNode(text));
+    checkbox.appendChild(label);
+    return checkbox;
+  }
+
+  /**
+   * Initialise component.
+   */
+  init() {
+    const containerClasses = [
+      'ecl-select__container',
+      'ecl-select__container--m',
+    ];
+    this.textDefault = this.element.getAttribute(this.defaultTextAttribute);
+    this.textSearch = this.element.getAttribute(this.searchTextAttribute);
+    this.textSelectAll = this.element.getAttribute(this.selectAllTextAttribute);
+
+    this.select = queryOne(this.selectMultipleSelector);
+
+    if (this.select.nextSibling.classList.contains('ecl-select__icon')) {
+      this.selectIcon = this.select.nextSibling;
+    }
+
+    this.selectMultiple = document.createElement('div');
+    this.selectMultiple.classList.add('ecl-select__multiple');
+    this.selectMultiple.addEventListener(
+      'click',
+      this.handleClickSelectMultiple
+    );
+
+    this.inputContainer = document.createElement('div');
+    this.inputContainer.classList.add(...containerClasses);
+    this.selectMultiple.appendChild(this.inputContainer);
+
+    this.input = document.createElement('input');
+    this.input.classList.add('ecl-select', 'ecl-select__multiple-toggle');
+    this.input.setAttribute('type', 'text');
+    this.input.setAttribute('placeholder', this.textDefault || '');
+    this.input.setAttribute('readonly', true);
+    this.inputContainer.appendChild(this.input);
+    this.inputContainer.appendChild(this.selectIcon);
+
+    this.searchContainer = document.createElement('div');
+    this.searchContainer.style.display = 'none';
+    this.searchContainer.classList.add(
+      'ecl-select__multiple-dropdown',
+      ...containerClasses
+    );
+    this.selectMultiple.appendChild(this.searchContainer);
+
+    this.search = document.createElement('input');
+    this.search.classList.add('ecl-text-input', 'ecl-text-input--m');
+    this.search.setAttribute('type', 'text');
+    this.search.setAttribute('placeholder', this.textSearch || '');
+    this.searchContainer.appendChild(this.search);
+
+    const selectAll = this.createCheckbox(
+      'all',
+      this.textSelectAll,
+      this.selectMultipleId
+    );
+    selectAll.addEventListener('click', this.handleClickSelectAll);
+    this.searchContainer.appendChild(selectAll);
```

Comment:
shoudn't it be selectContainer instead of searchContainer?

Reply:
I double-checked, the markup at the moment is matching the one from the other pull request with [reference markup](https://5ecb7eefd2193ec6ca8e002c--europa-component-library.netlify.app/playground/ec/iframe.html?id=components-forms-select--multiple) where `ecl-select__multiple-dropdown` contains search input and checkboxes which is named searchContainer in the JS implementation